### PR TITLE
unit 1 assignment

### PR DESCRIPTION
**Pull request by ntambe003**
@twang I tried to performing the method described in class, wherein a distinct function is called twice to repeat a message. It reported two errors that I didn't understand.

### CODE REVIEW

File: unit1.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+public class Mantra {
+    public static void main(String[] args) {
+        bigboi();
+        System.out.println();
+        bigboi();
+    }
+}
+        
+public class bigboi {
```
Comment (karenlei09) [inline unit1.java]: You are making bigboi a class here, but don't you want it to be a method that you can call?

File: unit1.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    public static void main() {
```
Comment (karenlei09) [inline unit1.java]: Why do you have two main methods? Can you rewrite it with one?

File: unit1.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
+public class Mantra {
+    public static void main(String[] args) {
```
Comment (karenlei09) [inline unit1.java]: Also, this is a style thing, but I would put the bigboi method before the main method; it just makes it clearer when you call it in the main method (It still works this way though)

**Comment by twang** [comment]
Thanks @ntambe003 and @karenlei09!

**Pull request closed without merging**